README.md

```diff
@@ -65,7 +67,7 @@ The command line arguments available from the terminal are also available from t
 
 | Terminal   |      Python      |
 |----------|-------------|
-| --input |  inFile |
+| --input |  input |
 | --output |  outFile |
```

Comment:
changing from `inFile` to `input` is fine, it is more consistent. However we should also do the same for `outFile` and `output`.